case_studies/strong_lensing/run_sim.sh

```diff
@@ -136,36 +140,42 @@ for i in $(seq 1 $num_files); do
         exit 1
     fi
 
-    echo "Running python file $PYTHON_SCRIPT_3 to stack all images..." &>> $LOG_FILE
-    python "$PYTHON_SCRIPT_3" "$OUTPUT_DIR" &>> $LOG_FILE
+    echo "Running python file $PYTHON_SCRIPT_3 to stack all images..."
+    python "$PYTHON_SCRIPT_3" "$OUTPUT_DIR"
 
-    # Fourht Python script - Converting Text Catalog to Fits
+    # Fourth Python script - Converting Text Catalog to Fits
     PYTHON_SCRIPT_4="convert_catalog.py"
     # Check if Python scripts exist
     if [ ! -f "$PYTHON_SCRIPT_4" ]; then
         echo "Python script $PYTHON_SCRIPT_4 not found!"
         exit 1
     fi
 
-    echo "Running python file $PYTHON_SCRIPT_4 to convert text catalog..." &>> $LOG_FILE
-    python "$PYTHON_SCRIPT_4" "$OUTPUT_DIR" &>> $LOG_FILE
+    echo "Running python file $PYTHON_SCRIPT_4 to convert text catalog..."
+    python "$PYTHON_SCRIPT_4" "$OUTPUT_DIR"
 
-    # Run Galsim to produce final image  
-    echo "Running galsim with $IMAGE_CONFIG_FILE..." &>> $LOG_FILE
-    galsim "$IMAGE_CONFIG_FILE" variables.output_dir="$FINAL_OUTPUT_DIR" variables.nobjects="$num_galaxies" variables.image_size="$img_size" &>> $LOG_FILE
+    # Run Galsim to produce final image
+    echo "Running galsim with $IMAGE_CONFIG_FILE..."
+    galsim "$IMAGE_CONFIG_FILE" variables.output_dir="$FINAL_OUTPUT_DIR" variables.nobjects="$num_galaxies" variables.image_size="$img_size"
 
     mv "$FINAL_OUTPUT_DIR/image.fits" "$FINAL_OUTPUT_DIR/image${i}.fits"
 
     OPEN_FITS="open_fits.py"
-    python "$OPEN_FITS" "$i" "$FINAL_OUTPUT_DIR" &>> $LOG_FILE
+
+    python "$OPEN_FITS" "$i" "$FINAL_OUTPUT_DIR"
 
     rm -r data/images
-    rm data/catalog.fits data/catalog.txt data/combined_images.fits
+    rm data/catalog.fits data/combined_images.fits
+    sed -i '$d' data/catalog.txt
+    mv data/catalog.txt $CATALOG_DIR/image${i}.txt
+
 done
 
 # Move all image pngs to separate folder
 mkdir $FINAL_OUTPUT_DIR/images
+mkdir $FINAL_OUTPUT_DIR/data
+
 mv $FINAL_OUTPUT_DIR/*.png $FINAL_OUTPUT_DIR/images/
+mv $FINAL_OUTPUT_DIR/*.fits $FINAL_OUTPUT_DIR/data/
 
 echo "Data generation completed."
-echo "Data generation ended."&>> $LOG_FILE
```

Comment:
Lack of a trailing space can lead to merge conflicts. Have you run `pre-commit install` within the bliss directory?

Reply:
I do have the pre-commit checks installed, and they do run before I commit, so I'm also unsure why there would be merge conflicts for that reason.